File: abc/abc327/a.cs
// https://atcoder.jp/contests/abc327/submissions/47345323

class A
{
  static void Main()
  {
    int n = int.Parse(Console.ReadLine());
    string s = Console.ReadLine();

    for(int i = 1; i < s.Length; i++)
    {
      if(s[i - 1] == 'a' && s[i] == 'b' || s[i - 1] == 'b' && s[i] == 'a')
      {
        Console.Write("Yes");
        return;
      }
    }

    Console.Write("No");
  }
}

File: abc/abc327/a.py
# https://atcoder.jp/contests/abc327/submissions/47415290

n = int(input())
s = input()

for i in range(1, n):
  if s[i - 1 : i + 1] in {'ab', 'ba'}:
    print('Yes')
    exit()

print('No')

